### ISSUE TITLE
1PM Mini Gen3 not working with PIR sensor

### ISSUE THREAD

**Issue by myhandsarecold** [mongoose-os-apps/shelly-homekit#1562]
I’m experiencing an issue where my PIR sensor isn’t triggering the Shelly. Both the Shelly and the PIR are brand new and I’ve verified they otherwise work. 

With the PIR sensor in the off position I’m reading ~120V across SW and N on the Shelly. When the PIR is on I’m reading ~242V across SW and N. However the switch status doesn’t change to “on” and therefore, in both scenarios, the voltage across O and N is ~1V

Conversely, if I replace the PIR with a physical switch and set it to off the voltage across SW and N is ~5 and when I flick the switch on it jumps to ~240, the Shelly state is turned on, and the voltage across O and N is ~240.

It’s possible I have a junk PIR sensor but it leads me to believe the threshold for triggering the Shelly state must be less than 120V (if I set initial state to “Input” and the Shelly boots with the PIR connected then the Shelly will initially turn on), or the delta must be greater than 120V.

Is this correct and can it be exposed as an advanced option or can you point me to the section of code where this is managed so I may fork?

**Comment by myhandsarecold** [comment]
Thanks @markirb. Still happens on original Shelly firmware.

I since found a support article which would seem to align with my experience: https://support.shelly.cloud/en/support/solutions/articles/103000221548-fixing-uncompatible-pir-sensors-with-shelly-devices.

To me it feels like the voltage threshold "should" be configurable. But then again presumably Shelly would have provided a firmware update instead of suggesting to use a resistor.

In any case feel free to close this ticket as I've decided to return the Shelly.

**Comment by myhandsarecold** [comment]
Thanks @markirb. Still happens on original Shelly firmware.

I since found a support article which would seem to align with my experience: https://support.shelly.cloud/en/support/solutions/articles/103000221548-fixing-uncompatible-pir-sensors-with-shelly-devices.

To me it feels like the voltage threshold "should" be configurable. But then again presumably Shelly would have provided a firmware update instead of suggesting to use a resistor.

I will close this ticket as I've decided to return the Shelly.
